app/src/main/java/com/android/voyageur/ui/overview/AddTrip.kt

```diff
@@ -419,7 +423,15 @@ fun AddTripScreen(
                             Text("Tourism", modifier = Modifier.padding(start = 2.dp))
                           }
                     }
-
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically) {
+                      Text("Make public", modifier = Modifier.padding(end = 16.dp))
```

Comment:
Instead of “Make public,” consider a more descriptive label like “Allow others to discover this trip” to make the functionality explicit.

Reply:
I was thinking about this, but I believe this will make the text wrap to 2 lines instead of just 1 and would look a bit weird. But I can give it a try for sure, thanks for the review.

Reply:
I updated this and made it a string resource as well